ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -899,19 +947,14 @@ - (void)handleOpenDocumentMethod:(NSDictionary<NSString *, id> *)arguments resul
         password = (NSString *)passwordValue;
     }
     
-    // Create and wrap a tabbed controller in a navigation controller.
-    self.tabbedDocumentViewController = [[PTTabbedDocumentViewController alloc] init];
-    self.tabbedDocumentViewController.delegate = self;
-    self.tabbedDocumentViewController.tabsEnabled = NO;
-    self.tabbedDocumentViewController.viewControllerClass = [PTFlutterDocumentController class];
-    
-    PTNavigationController *navigationController = [[PTNavigationController alloc] initWithRootViewController:self.tabbedDocumentViewController];
-    
     NSString* config = arguments[PTConfigArgumentKey];
     self.config = config;
     
-    [[self class] configureTabbedDocumentViewController:self.tabbedDocumentViewController
-                                             withConfig:config];
+    if (!self.tabbedDocumentViewController) {
+        [self initTabbedDocumentViewController];
+    }
+    
+    [PdftronFlutterPlugin configureTabbedDocumentViewController:self.tabbedDocumentViewController withConfig:config];
     
     // Open a file URL.
     NSURL *fileURL = [[NSBundle mainBundle] URLForResource:document withExtension:@"pdf"];
```

Comment:
This looks wrong to me. Is `document` just a filename, not a full path? Because this will only open documents with the filename contained in `document`, that are in the main bundle, and only PDFs (even though we support office docs, images, etc.).

Reply:
I think this is outside the scope of this PR. We can add an issue and work on it in another PR.